### ISSUE TITLE
"flake8 -q" doesn't list files with errors

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#74]
In GitLab by @jodal on Jan 28, 2015, 06:35

`flake8 --help` states:

>   -q, --quiet           report only file names, or nothing with -qq

I've tested both flake8 2.3.0 and latest from Git, and I fail to get any output at all with `flake8 -q`. I'm expecting it to print the filenames with errors, without any details about line numbers and error types.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Jan 28, 2015, 10:48

I was able to see the correct behaviour with pep8 1.5.6 and flake8 2.1.0. I have yet to test pep8 1.5.7 independently (or other versions of flake8 after 2.1.0). This option is entirely handled by pep8 (if I remember correctly) but flake8 could be altering the behaviour somewhere subtly

**Comment by asottile** [comment]
In GitLab by @jodal on Jan 28, 2015, 11:13

I get the correct behavior with flake8 2.1.0, using latest version of all the deps:

```
$ flake8 --version
2.1.0 (pep8: 1.5.7, pyflakes: 0.8.1, mccabe: 0.3) CPython 2.7.8 on Linux
$ flake8 -q .
foo.py
$
```

I've tested 2.2.0, 2.2.5 and 2.3.0, and all are broken in the same way, using the same dependencies as with 2.1.0:

```
$ flake8 --version
2.2.0 (pep8: 1.5.7, pyflakes: 0.8.1, mccabe: 0.3) CPython 2.7.8 on Linux
$ flake8 -q .
$
```

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Jan 28, 2015, 11:14

Interesting. Thanks for that @jodal

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Jan 28, 2015, 11:19

```
$ flake8 --version
2.3.0 (pep8: 1.5.7, pyflakes: 0.8.1, mccabe: 0.3) CPython 2.7.9 on Darwin
$  flake8 --jobs=1 -q requests/*.py
requests/__init__.py
requests/adapters.py
requests/api.py
requests/auth.py
requests/compat.py
requests/cookies.py
requests/models.py
requests/sessions.py
requests/status_codes.py
requests/structures.py
requests/utils.py
```

This would seem to be another consequence of multiprocessing. I'll have to track it down and figure out why this is happening later.

**Comment by asottile** [comment]
In GitLab by @jodal on Jan 28, 2015, 11:24

Indeed, it works for me too with `--jobs=1`.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Feb 22, 2015, 09:44

Hey @jodal feel free to check out !21 for a fix to this issue.

Sorry it took ~1 month.

Cheers!
Ian

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Feb 22, 2015, 10:05

Status changed to closed by commit d6a5df1e583a99673d98cb6e70ec91223475547a
